File: server/src/main/java/vlgo/server/response/LoginResponse.java
package vlgo.server.response;

public class LoginResponse<T> extends ResponseForm<T> {
    private String access_token;

    public LoginResponse(Integer code, String message, T data, String jwt) {
        super(code, message, data);
        this.access_token = jwt;
    }

    public void getAccessToken(String token){
        this.access_token = token;
    }

    public String getAccessToken() {
        return this.access_token;
    }

    @Override
    public boolean equals(Object o) {
        return super.equals(o);
    }

    @Override
    public int hashCode() {
        return super.hashCode();
    }
}

File: frontend/src/common/LoginResponse.ts
import BaseResponse from './BaseResponse';
export default class LoginResponse<T> extends BaseResponse<T> {
    public accessToken = ""

    constructor(code: number, message: string, data: T, datas: T[], accessToken: string){
        super(code, message, data, datas)
        this.accessToken = accessToken;
    }
}